ape_starknet/accounts/__init__.py

```diff
@@ -349,6 +451,18 @@ def delete_account(
         networks: Optional[Union[str, List[str]]] = None,
         leave_unlocked: Optional[bool] = None,
     ):
+        """
+        Delete an account.
+
+        Args:
+            alias (str): A shortened-name for quicker access to the account.
+            address (Optional[Union[AddressType, int]]):
+                The address of the account to be deleted Defaults to `None`.
+            networks (Optional[Union[str, List[str]]]):
+                The network(s) for the accounts to be deleted from. Defaults to `None`.
+            leave_unlocked (bool):
+                Option to leave account unlocked for future interactions. Defaults to `None`.
```

Comment:
In doc-strs, you have to use 2 tick marks.

```suggestion
                Option to leave account unlocked for future interactions. Defaults to ``None``.
```

- [ ] Replace all single-tics with double tics in doc-strs